pkgs/tools/networking/hysteria/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib,
+  fetchFromGitHub,
+  buildGoModule,
+}:
+buildGoModule rec {
+  pname = "hysteria";
+  version = "1.1.0";
+
+  src = fetchFromGitHub {
+    owner = "HyNetwork";
+    repo = "${pname}";
+    rev = "v${version}";
+    sha256 = "sha256-V+umf7+qRANSNsMrU1Vij3ni6ayq/d41xSy3o+7sEHQ=";
+  };
+
+  vendorSha256 = "sha256-oxCZ4+E3kffHr8ca9BKCSYcSWQ8jwpzrFs0fvCvZyJE=";
+  proxyVendor = true;
+
+  doCheck = false;
```

Comment:
Maybe you should add an explanation for why you skipped the check.

Reply:
Reason added.